apps/client/src/pages/home/Home.page.tsx

```diff
@@ -48,7 +63,21 @@ export function Home(): ReactElement {
           'flex flex-col justify-center items-center',
           'w-[500px] gap-8',
         )}>
-        <img src={logo} className='-mb-16' alt='' />
+        {/* <img src={logo} className='-mb-16' alt='' /> */}
```

Comment:
Commented code

Reply:
LGTM except for the above issue. 

Also, @supunTE what would be the possibility of having SVGs instead of the PNGs for the Logos

Reply:
> Commented code

This was removed from later commits. 
See the most recent layout of the file - https://github.com/the-ai-team/razor/pull/73/files#diff-7d1aba9f93cbe79697548cd94aedb53cf93f4973bc3db4a9b94e14378a5bb765

Reply:
> Also, @supunTE what would be the possibility of having SVGs instead of the PNGs for the Logos

Currently, we're using SVGs for the logo and the small car icon.